WapoJS/src/js_eval.rs

```diff
@@ -71,25 +94,33 @@ fn parse_args(args: impl Iterator<Item = String>) -> Result<Args> {
         print_usage();
         bail!("no script file provided");
     }
+    if worker_secret.is_none() {
```

Comment:
I feel it inconvenient to use wapojs to do tests, if the `--worker-secret` is a required option. Should we give it a default value?

Reply:
Since it's only required for native build, how about give a default value and showing a warning message?

Reply:
> Since it's only required for native build, how about give a default value and showing a warning message?

Good idea.